### ISSUE TITLE
Support for .dll

### ISSUE THREAD

**Issue by myselfhimself** [vinayak-mehta/pedep#2]
Hello,
do you know it this tool supports Python built .dll files instead of .pyd files ?
If yes, it could be nice to write it down in the README file.
:)

**Comment by vinayak-mehta** [comment]
Yes! I think it should work on any DLL or PYD, because they are PE files.

**Comment by vinayak-mehta** [comment]
> If yes, it could be nice to write it down in the README file.

Would you like to raise a PR to do that? :)

**Comment by myselfhimself** [comment]
I meant the README file of a new separate repository, such as 
If you create that other repository, I can commit some pull requests for such janitor work for sure, no problem.

**Comment by myselfhimself** [comment]
OOps mixing issues

**Comment by myselfhimself** [comment]
Done
